x-pack/plugins/security_solution/public/management/cypress/e2e/artifacts.cy.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isEmpty } from 'lodash';
+import {
+  ENDPOINT_ARTIFACT_LIST_IDS,
+  EXCEPTION_LIST_URL,
+} from '@kbn/securitysolution-list-constants';
+import { BASE_ENDPOINT_ROUTE } from '../../../../common/endpoint/constants';
+import { login, loginWithRole, ROLE } from '../tasks/login';
+
+import { type FormAction, getArtifactsListTestsData } from './mocks/artifacts_page';
+import { runEndpointLoaderScript } from '../tasks/run_endpoint_loader';
+
+const removeAllArtifacts = () => {
+  for (const listId of ENDPOINT_ARTIFACT_LIST_IDS) {
+    cy.request({
```

Comment:
should you `await` this?

I also noticed that `removeAllArtifacts()` is not an async function.

Reply:
It is not possible to `await` Cypress `request` method, it yields the response instead. For those you want to get the result, you can use `then` as is used in `loadEndpointDataForEventFiltersIfNeeded`.